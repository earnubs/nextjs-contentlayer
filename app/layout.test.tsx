import {render, screen} from '@testing-library/react'
import RootLayout from './layout'

describe("<RootLayout />", () => {
  test("renders as expected", () => {
    const { container } = render(<RootLayout>Test</RootLayout>)

    expect(container.firstChild).toMatchSnapshot();
  })
})
