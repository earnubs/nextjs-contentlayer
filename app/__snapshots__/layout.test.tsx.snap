// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RootLayout /> renders as expected 1`] = `
<html
  lang="en"
>
  <body>
    Test
  </body>
</html>
`;
